publisher-sdk/src/main/java/com/criteo/publisher/BidResponse.java

```diff
@@ -17,73 +17,46 @@
 package com.criteo.publisher;
 
 import androidx.annotation.Keep;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import com.criteo.publisher.model.AbstractTokenValue;
```

Comment:
FAR: what do you think about renaming `AbstractTokenValue` to `TokenValue`?

Reply:
This class is removed in 7ec164d